ui-components/src/page_components/listing/listing_sidebar/events/DownloadLotteryResults.tsx

```diff
@@ -1,21 +1,23 @@
 import * as React from "react"
-import { ListingEvent } from "@bloom-housing/backend-core/types"
+import { ListingEvent, ListingEventType } from "@bloom-housing/backend-core/types"
```

Comment:
Just as a note, since this component already has backend type dependencies and can be handled separately, but like the comment in helpers/pdfs, we want to move away from Bloom core dependencies.